V4_README.md

```diff
@@ -0,0 +1,93 @@
+Python bindings for the Luminoso client API
+===========================================
+
+This package contains Python code for interacting with a Luminoso text
+processing server through its REST API.
+
+Important note: API version and client version
+----------------------------------------------
+
+This page covers the client that connects to the v4 API; this client is the
+object named `luminoso_api.V4LuminosoClient`, which is an alias for
+`luminoso_api.v4_client.LuminosoClient`.
+
+Please note that the v4 client can only be used to connect to v4 endpoints,
+which do not include any science endpoints for building or interacting with
+projects.  This client can only be used for user and account management.
+
+Using this client
+=================
+
+In this code, instead of having to authenticate each request separately,
+you make a "session" object that keeps track of your login information,
+and call methods on it that will be properly authenticated.
+
+Installation
+---------------
+This client API is designed to be used with Python 2.6, 2.7, 3.3, or 3.4.
+
+You can download and install it using a Python package manager:
+
+    pip install luminoso-api
+
+or
+
+    easy_install luminoso-api
+
+Or you can download this repository and install it the usual way:
+
+    python setup.py install
+
+If you are installing into the main Python environment on a Mac or Unix
+system, you will probably need to prefix those commands with `sudo` and
+enter your password, as in `sudo python setup.py install`.
+
+Getting started
+---------------
+You interact with the API using a LuminosoClient object, which sends HTTP
+requests to URLs starting with a given path, and keeps track of your
+authentication information.
+
+You can connect using a username and password:
+
+```python
+>>> from luminoso_api import LuminosoClient
```

Comment:
I believe this is now wrong -- @moss, can you confirm and if so I'll push the change?

Reply:
Yes, that's wrong.